tox.ini

```diff
@@ -1,3 +1,22 @@
+[tox]
+envlist = py27,py34,style
+
+[testenv]
+install_command = pip install -U {opts} {packages}
+passenv = AWS_*
+deps = -r{toxinidir}/requirements.txt
+       .[test]
+commands=
+    python -c "import sys;print('\nPYTHON VERSION\n%s\n' % sys.version)"
+    py.test --verbose --color auto
+
+[testenv:style]
+deps =
+      .[style]
+basepython = python2.7
+commands =
+    flake8 lambda_uploader setup.py --statistics
```

Comment:
Not in this run. There are a lot of changes, I'll make a separate PR for this to keep this one reasonable.